app/survey.tsx

```
@@ -0,0 +1,656 @@
+          case 'MULTIPLE_CHOICE':
+            // 배열에서 __OTHER__를 other로 변환
+            const convertedAnswers = Array.isArray(answer) 
+              ? answer.map(a => a === '__OTHER__' ? 'other' : a)
+              : answer;
+            responseData = {
+              questionId: question.questionId,
+              answers: convertedAnswers,
+              otherText: otherText,
+              note: null
+            };
+            break;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)